azurelinuxagent/common/osutil/default.py

```diff
@@ -1248,7 +1248,7 @@ def get_NIC_state(self):
         """
         state = {}
 
-        status, output = shellutil.run_get_output("ip -a -d -o link")
+        status, output = shellutil.run_get_output("/sbin/ip -a -d -o link")
```

Comment:
I don't think we use full path.  I would be leery of hard coding a path because it prevents customers from overriding 

I would either disable this test in Travis, or modify Travis' environment to include ip.  You may have to do the former.